src/Http/Routing/src/RouteOptions.cs

```diff
@@ -49,6 +61,7 @@ public class RouteOptions
         public bool SuppressCheckForUnhandledSecurityMetadata { get; set; }
 
         private IDictionary<string, Type> _constraintTypeMap = GetDefaultConstraintMap();
+        private ICollection<EndpointDataSource> _endpointDataSources;
```

Comment:
Can you just move these fields to the top? This isn't the way our guidelines be.